<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFusionLayoutWidgetBase</class>
 <widget class="QWidget" name="QFusionLayoutWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="udg::ItemMenu" name="m_item2x1First">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x1_1_2"/>
        </item>
        <item row="0" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x1_1_1"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_label2x1First">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>2x1 CT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="udg::ItemMenu" name="m_item2x1Second">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x1_2_1"/>
        </item>
        <item row="0" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x1_2_2"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_label2x1Second">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>2x1 PT/NM</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="udg::ItemMenu" name="m_item3x1">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon3x1_1"/>
        </item>
        <item row="0" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon3x1_2"/>
        </item>
        <item row="0" column="2">
         <widget class="udg::GridIcon" name="m_gridIcon3x1_3"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_label3x1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>3x1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="udg::ItemMenu" name="m_item2x3First">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_1_1"/>
        </item>
        <item row="0" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_1_2"/>
        </item>
        <item row="1" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_1_3"/>
        </item>
        <item row="1" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_1_4"/>
        </item>
        <item row="2" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_1_5"/>
        </item>
        <item row="2" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_1_6"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_label2x3First">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>2x3 CT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="udg::ItemMenu" name="m_item2x3Second">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_2_1"/>
        </item>
        <item row="0" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_2_2"/>
        </item>
        <item row="1" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_2_3"/>
        </item>
        <item row="1" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_2_4"/>
        </item>
        <item row="2" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_2_5"/>
        </item>
        <item row="2" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon2x3_2_6"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_label2x3Second">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>2x3 PT/NM</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="udg::ItemMenu" name="m_item3x3">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_1"/>
        </item>
        <item row="0" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_2"/>
        </item>
        <item row="0" column="2">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_3"/>
        </item>
        <item row="1" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_4"/>
        </item>
        <item row="1" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_5"/>
        </item>
        <item row="1" column="2">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_6"/>
        </item>
        <item row="2" column="0">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_7"/>
        </item>
        <item row="2" column="1">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_8"/>
        </item>
        <item row="2" column="2">
         <widget class="udg::GridIcon" name="m_gridIcon3x3_9"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_label3x3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>3x3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::ItemMenu</class>
   <extends>QFrame</extends>
   <header>itemmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>udg::GridIcon</class>
   <extends>QFrame</extends>
   <header>gridicon.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
